src/providers/krb5/krb5_child.c

```diff
@@ -773,6 +773,11 @@ static krb5_error_code answer_pkinit(krb5_context ctx,
                   "krb5_responder_set_answer failed.\n");
         }
 
+        goto done;
+    } else {
+        DEBUG(SSSDBG_MINOR_FAILURE, "Unexpected authentication token type [%s]\n",
+              sss_authtok_type_to_str(sss_authtok_get_type(kr->pd->authtok)));
+        kerr = EAGAIN;
         goto done;
     }
 
```

Comment:
What is the purpose of `kerr = EOK;` below?

Reply:
I didn't modify that code, I guess it can be removed?

Reply:
Previously this assignment could be reached, now it can't.

Reply:
So it will yield a covscan error at least.

Reply:
Removed `kerr = EOK`